components/statuslight/index.css

```diff
@@ -13,7 +13,9 @@ governing permissions and limitations under the License.
 @import '../commons/index.css';
 
 :root {
-  --spectrum-statuslight-padding-y: var(--spectrum-global-dimension-size-75);
+  /* Hardcoded for wrapping study.
+  Will be a DNA token in https://jira.corp.adobe.com/browse/SDS-4466 */
+  --spectrum-statuslight-padding-y: var(--spectrum-global-dimension-size-65);
```

Comment:
This was already fixed and verified in #426. What changed? I see no additional comments on that issue.

Reply:
@lazd @bernhard-adobe I ran a VR test to help you see the difference. https://spectrum-visual-regression.ci.corp.adobe.com/job/spectrum-css-vr-test-asset/139/artifact/spectrum-css/backstop_data/html_report/index.html

Reply:
A VR test like only takes less than 10mins on jenkins. I encourage you all run at least one to see the change for this kind of PR.

Reply:
Thanks, @jianliao. I'm wondering why this needed adjustment after a verified fix was already in.